packages/blocks/src/page-block/edgeless/edgeless-page-block.ts

```diff
@@ -802,6 +805,23 @@ export class EdgelessPageBlockComponent
     }
   }
 
+  getElementsBound(): IBound | null {
+    const bounds = [];
+
+    this.frames.forEach(frame => {
+      const frameXYWH = deserializeXYWH(frame.xywh);
```

Comment:
here, we now have Bound.deserializeXYWH, if you like, you can use it later

Reply:
Feel free to open minor refactoring PRs!